cmd/kops/validate.go

```diff
@@ -31,7 +31,7 @@ var (
 
 	1. All k8s masters are running and have "Ready" status.
 	2. All k8s nodes are running and have "Ready" status.
-	3. Componentstatues returns healthy for all components.
+	3. Component statues returns healthy for all components.
```

Comment:
I think this is a typo in the original (as well as the missing space) - it should be "Component status returns healthy..."

Reply:
Fixed again. good!